integration-tests/src/test/java/io/druid/tests/hadoop/ITHadoopIndexTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.tests.hadoop;
+
+import com.google.common.base.Throwables;
+import com.google.inject.Inject;
+import com.metamx.common.ISE;
+import com.metamx.common.logger.Logger;
+import io.druid.indexing.common.TaskStatus;
+import io.druid.testing.IntegrationTestingConfig;
+import io.druid.testing.guice.DruidTestModuleFactory;
+import io.druid.testing.utils.RetryUtil;
+import io.druid.tests.indexer.AbstractIndexerTest;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Guice;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.Callable;
+
+@Guice(moduleFactory = DruidTestModuleFactory.class)
+public class ITHadoopIndexTest extends AbstractIndexerTest
+{
+  private static final Logger LOG = new Logger(ITHadoopIndexTest.class);
+  private static String BATCH_TASK = "/hadoop/batch_hadoop_indexer.json";
+  private static String BATCH_QUERIES_RESOURCE = "/hadoop/batch_hadoop_queries.json";
+  private static String BATCH_DATASOURCE = "batchHadoop";
+  private static boolean dataLoaded = false;
+
+  @Inject
+  private IntegrationTestingConfig config;
+
+  @BeforeClass
+  public void beforeClass()
+  {
+    loadData(config.getProperty ("hadoopTestDir") + "/batchHadoop1");
+    dataLoaded = true;
```

Comment:
not sure why this variable is needed, wouldn't test fail if loadData threw some RuntimeException on failure to loading data?